ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -62,9 +64,6 @@ private[appendonlydao] final class TransactionsReader(
   // This significantly improves the performance of the transaction service.
   private val outputStreamBufferSize = 128
 
-  // TODO: make this parameter configurable
-  private val ContractStateEventsStreamParallelismLevel = 4
-
   private val TransactionEventsFetchParallelism = 8
```

Comment:
Very noble PR. Can you please reuse the config parameter for this value (`TransactionEventsFetchParallelism`) as well?
Context: These two are never to be used concurrently (i.e. it depends on the in-memory fan-out toggle which one is used), but they both can leverage the CLI config param

Reply:
Those two have different default values and a slightly different meaning, I would be careful with reusing the same config parameter.

Are you sure you want this parameters default value to be linked to the default value of the other parameter?

Reply:
Yeah, the different default values are random. 
1. wrt to the meaning, they both specify the parallelism used in the fetching/decoding with the only difference being that `getContractStateEvents` doesn't use parallel fetch but only parallel decode. 
2. `getContractStateEvents` which uses `ContractStateEventsStreamParallelismLevel` is interchangeable in usage with `getTransactionLogUpdates` which uses `TransactionEventsFetchParallelism`: they both are internal streams meant for populating the LAPI caches/buffers. When in-memory fan-out is enabled, the later used. Without in-memory fan-out `getContractStateEvents` is used

Reply:
@kamil-da Are you ok with changing the default of `ContractStateEventsStreamParallelismLevel` to 8?

Reply:
Yes, I think it's ok.

Reply:
@kamil-da @tudor-da I have updated the PR according to the above discussion.